templates/main/index.html.ep

```diff
@@ -19,7 +19,6 @@
     </div>
 </div>
 
-<a name="build-results"/>
```

Comment:
I think it is best to use an ID and don't increase indendation. Then the indendation is messed, but it is better than using deprecated HTML. (In this case the HTML is even broken which can cause further problems eg. when using DOM and CSS selectors).